k8s/charts/orders-api/values.yaml

```diff
@@ -0,0 +1,25 @@
+kubernetesClusterDomain: cluster.local
+ordersApi:
+  ordersApi:
```

Comment:
i dont like how `helmify` did this double nesting, i'll probably go back and clean that up

Reply:
ah that's gross